src/app/projects/projects-table/projects-table.component.html

```diff
@@ -0,0 +1,44 @@
+<app-menubar></app-menubar>
+
+<div class="ui-g" id="projects-table">
+  <div class="ui-g-9">
+    <h5>Dostępne projekty</h5>
+    <p-table>
+      <ng-template pTemplate="header">
+        <tr>
+          <th>Lp</th>
+          <th>Nazwa</th>
+          <th>Admin</th>
+          <th>Link</th>
+        </tr>
+        <tr *ngFor="let project of projects">
+          <td>{{project.id}}</td>
+          <td>{{project.name}}</td>
+          <td>{{project.admin}}</td>
+          <td><p-button label="" icon="pi pi-arrow-right" [routerLink]="['/example']"></p-button></td>
+        </tr>
+      </ng-template>
+
+    </p-table>
+    <hr>
+    <p>W tabeli docelowo znajdzie się efekt zapytania "GET /projects"</p>
+  </div>
+
+  <div class="ui-g-3">
+    <h5>Nowy projekt</h5>
+
+    <p>Nazwa:</p>
+    <input type="text" pInputText [(ngModel)]="newName"/>
+
+    <p class="first">Współczynnik początkowy: {{newCoef}}</p>
+    <p-slider [(ngModel)]="newCoef" [style]="{'width':'15em'}"
+              [min]="1" [max]="3" [step]="0.05"></p-slider>
+
+    <p></p>
```

Comment:
nie mam tego jak sprawdzić, ale czy to jest konieczne?

Reply:
o pusty akapit Ci chodzi?

Reply:
wydaje mi się że opakowanie w diva tego co jest nad i w drugiego diva tego co jest pod powinno załatwić sprawę (ewentualnie jakieś stylowanie będzie jeszcze potrzebne)